exercises/grep/canonical-data.json

```diff
@@ -332,6 +363,54 @@
             "files": ["iliad.txt", "midsummer-night.txt", "paradise-lost.txt"]
           },
           "expected": []
+        },
+        {
+          "description": "Multiple files, several matches, line number and file flags",
+          "property": "grep",
+          "input": {
+            "pattern": "who",
+            "flags": ["-n", "-l"],
+            "files": ["iliad.txt", "midsummer-night.txt", "paradise-lost.txt"]
+          },
+          "expected": [
+            "2:His wrath pernicious, who ten thousand woes",
+            "2:Of that Forbidden Tree, whose mortal tast",
+            "8:That Shepherd, who first taught the chosen Seed"
+          ]
+        },
+        {
+          "description": "Multiple files, several matches, inverted and match entire lines flags",
+          "property": "grep",
+          "input": {
+            "pattern": "Illustrious into Ades premature",
```

Comment:
needs a comma, otherwise it does not match the entire line, as in https://github.com/exercism/problem-specifications/pull/1356/files#r223276145

Reply:
Yes, I'll fix that one too.